packages/website/vite.config.ts

```diff
@@ -4,6 +4,11 @@ export default defineConfig(({ mode }) => {
   process.env = { ...process.env, ...loadEnv(mode, process.cwd()) };
   if (mode === 'development') {
     return {
+      optimizeDeps: {
+        include: [
+          // '@uidotdev/usehooks',
```

Comment:
I expected this would improve `npm run website:dev`, but it makes it worse (errors) if we remove `--with-ssr`.